website_sale_require_legal/README.rst

```diff
@@ -0,0 +1,112 @@
+=========================================
+Require accepting legal terms to checkout
+=========================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/13.0/website_sale_require_legal
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_require_legal
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module was written to extend the functionality of your website shop to
+support forcing the user to accept your legal advice, terms of use and privacy
+policy, and allow you to comply with some countries' laws.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
+
+* Install repository `OCA/website <https://github.com/OCA/website>`_.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Set up your legal pages.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Buy something from your website.
+
+Known issues / Roadmap
+======================
+
+* Shopping terms and conditions are accepted only on user registration or
+  address edition. So if those terms change after the user signed up, a
```

Comment:
With the new behavior, the checkbox is displayed in checkout step.

Reply:
@sergio-teruel , the Odoo (core) option shows the check box always just before paying, but what this point of the readme refers to is to show that box only if the terms have changed (perhaps to be shown before paying as well). I think they are two different things. And it seems to me that to cover that point (if we needed), we must add the functionality in this module, because the odoo option is not very configurable since it is a template designed to be activated and deactivated by the UI.

Reply:
What do you think?